### DISCUSSION TITLE
This plugin handles [Header()], [HideInAttribute], [Space] on the runtime inspector? Here is not working

### DISCUSSION THREAD

**Discussion by riqmariz** [yasirkula/UnityRuntimeInspector#26] [Q&A] [answered]
Hey there, if possible, I want to know if this plugin supports [Header()], [HideInAttribute]. [Space] on the runtime inspector. It is? Here i cannot make it display [Header] for example

**Comment by yasirkula** [comment, marked as answer]
Only HideInInspector, SerializeField, System.NonSerialized and Range attributes are supported.

**Reply by yasirkula** [reply]
But I agree that Header and Space attributes could be useful.

**Reply by riqmariz** [reply]
Thanks for the reply!! This plugin is saving me a lot of time!

**Reply by riqmariz** [reply]
I forgot to ask about [Tooltip()], I think it has support, because I saw a checkbox called "Show Tooltip" on the runtimeInspector. But for me is not working, when I stop my cursor on the variable, it shows just the name of the variable and not what was wrote on the tooltip box

**Reply by yasirkula** [reply]
Yes, tooltip just displays the name. It is useful when a long variable's name is obscured by its value; then you can hover over the variable to see its full name.

**Reply by riqmariz** [reply]
Got it, thanks
